sitegen/home.ts

```diff
@@ -138,6 +138,11 @@ van.add(document.body, Counter())
           td("An SPA framework add-on for ", VanJS()),
           td(Link("b-rad-c", "https://github.com/b-rad-c")),
         ),
+        tr(
+          td(Link("VanJS Feather", "https://thednp.github.io/vanjs-feather/")),
```

Comment:
Can you put some instructions on how to use your library in the website? If the github repo servers better for this purpose, you can put the link to the github repo here.

Reply:
That's the plan, to include instructions.

Reply:
Sure. I will merge this PR once the instructions are there.

Reply:
https://thednp.github.io/vanjs-feather/